### DISCUSSION TITLE
Just finished my first board.

### DISCUSSION THREAD

**Discussion by w7com** [picocomputer/community#106] [Q&A]
This sure takes me back to 1984ish when I was working for Wygant Scientific in Portland building telemarketing machines out of 6502s.
Happiness is a warm (soldering) gun, momma.  Thanks Rumble for a great evening.  Tomorrow I'll shove some code at it and see if I did it all correctly.  -Joe
![D0CE598B-31E3-4E47-B97E-7962F5A96F98_1_201_a](https://github.com/picocomputer/community/assets/153972288/1df53ff1-90c2-42ce-83f9-036cb15dee37)

**Comment by vruumllc** [comment]
Welcome!
